src/main.c

```diff
@@ -436,6 +442,19 @@ main
 #endif // NO_VIM_MAIN
 #endif // PROTO
 
+#if defined(EXITFREE) || defined(PROTO)
+    void
+free_vbuf(void)
+{
+    if (s_vbuf != NULL)
+    {
+	setvbuf(stdout, NULL, _IONBF, 0);
```

Comment:
> setvbuf(stdout, NULL, _IONBF, 0);

Is this call required? Because now you are introducing the same problem that this PR is fixing? Shouldn't `size` be larger than 1?

Reply:
Before freeing `s_vbuf` we should stop using it in `setvbuf()`.
So I use `_IONBF` (not `_IOLBF`) here to stop buffering.

Reply:
ah, thanks. Makes sense

Reply:
If you have installed MSVC, you should be able to find the source code of `setvbuf()` here:
`C:\Program Files (x86)\Windows Kits\10\Source\<BuildNumber>\ucrt\stdio\setvbuf.cpp`